float32[6] position
float32[2] gripper
float32[18] orientation
float32[6] vel_position
float32[2] vel_gripper
float32[6] vel_orientation
